randovania/data/json_data/prime1/Phazon Mines.txt

```diff
@@ -647,13 +647,18 @@ Asset id: 961790803
       All of the following:
           X-Ray Visor
           Any of the following:
+              Combat (Hypermode)
               Charge Beam and Have all Beams
               All of the following:
-                  Charge Beam and Combat (Beginner)
+                  Charge Beam and Combat (Beginner) and Normal Damage ≥ 200
                   Any of the following:
+                      # 1-Cycle Strats
                       Plasma Beam
-                      Missile ≥ 50 and Super Missile
+                      Missile ≥ 30 and Super Missile
               Power Bomb ≥ 3 and Morph Ball and Combat (Advanced)
+              Combat (Expert) and Normal Damage ≥ 300
+              Combat (Beginner) and Normal Damage ≥ 1000
```

Comment:
1000 seems excessive due to there still being the multiplier, but it's for chargeless so 🤷

Reply:
- NSJ makes it harder to jump over the attacks
- Chargeless makes it take like 4-5 cycles instead of 1-2
- Beamless means you have to dodge colored pirates the entire time

Reply:
Oh right the multiplier. I thought you were talking about suit reduction. I forgot that existed and we put in the minimum damage requirements, not the median. I'll change this.